example/ios/AirshipSample.xcodeproj/project.pbxproj

```diff
@@ -389,7 +356,7 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.urbanairship.richpush;
+				PRODUCT_BUNDLE_IDENTIFIER = com.urbanairship.goat;
```

Comment:
was this for testing? probably should revert this line and the one below

Reply:
Right, I forgot. I'll do it from Xcode, I had a bad experience last time editing the file manually lol